{"name": "test_loginFailMethod", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <AppiumFramwork.tests.LoginTest.LoginTest testMethod=test_loginFailMethod>\n\n    @pytest.mark.run(order=3)\n    def test_loginFailMethod(self):\n        cl.allureLogs(\"App Opened\")\n        self.gt.clickLoginBotton()\n        self.gt.enterEmail()\n        self.gt.enterPassword2()\n        self.gt.clickOnLoginB()\n>       self.gt.verifyAdminScreen()\n\nLoginTest.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <AppiumFramwork.pages.LoginPage.LoginPageTest object at 0x000001FA434EF5E0>\n\n    def verifyAdminScreen(self):\n        adminScreen = self.isDisplayed(self._pageTitle,\"text\")\n>       assert adminScreen == True\nE       AssertionError\n\n..\\pages\\LoginPage.py:44: AssertionError"}, "steps": [{"name": "App Opened", "status": "passed", "start": 1653634726212, "stop": 1653634726212}, {"name": "Click on Login Button", "status": "passed", "start": 1653634726844, "stop": 1653634726844}, {"name": "Entered email id", "status": "passed", "start": 1653634728278, "stop": 1653634728278}, {"name": "Entered Password", "status": "passed", "start": 1653634729345, "stop": 1653634729345}, {"name": "Clicked on Login Button in Login Screen", "status": "passed", "start": 1653634729890, "stop": 1653634729890}], "attachments": [{"name": "text", "source": "9e840ea2-eafa-4fad-919c-62b5689c510f-attachment.png", "type": "image/png"}, {"name": "text", "source": "5b7dba04-d6c8-4afc-96b8-62c809d12c1d-attachment.png", "type": "image/png"}, {"name": "log", "source": "1e5f5a59-14d7-456f-9b7f-1bcd143ad466-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "a5753dc9-a7e4-4c1f-9c8e-5d25774d0b37-attachment.txt", "type": "text/plain"}], "start": 1653634726212, "stop": 1653634755581, "uuid": "584a3f07-0e87-44c4-8e9e-582914727dc9", "historyId": "aeb6c84858cb897d59574054d46d09e3", "testCaseId": "1512c76b4b2d60f059541c0f73c45353", "fullName": "TestSuite.LoginTest#test_loginFailMethod", "labels": [{"name": "tag", "value": "run(order=3)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "suite", "value": "TestSuite"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "BTZVBK3"}, {"name": "thread", "value": "19644-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSuite"}]}